cliconfig/credentials.go

```diff
@@ -0,0 +1,101 @@
+package cliconfig
+
+import (
+	"github.com/docker/docker/cliconfig/credentials"
+	"github.com/docker/engine-api/types"
+)
+
+// GetCredentials loads the user credentials for a given registry server.
+func (c *ConfigFile) GetCredentials(serverAddress string) (types.AuthConfig, error) {
```

Comment:
Does it make sense architecturally to have the configuration file take on the role of saving credentials?